app/controllers/admin/export/document_controller.rb

```diff
@@ -1,8 +1,44 @@
+require "#{Rails.root}/lib/whitehall/exporters/documents_info_exporter"
+
 class Admin::Export::DocumentController < Admin::Export::BaseController
   self.responder = Api::Responder
 
   def show
     @document = Document.find(params[:id])
     respond_with DocumentExportPresenter.new(@document)
   end
+
+  def index
+    result_set = DocumentsInfoExporter.new(paginated_document_ids).call
+
+    respond_with(
+      documents: result_set,
+      page_number: page_number,
+      page_count: result_set.count
+    )
+  end
+
+  private
+
+  def paginated_document_ids
+    org_id = Organisation.find_by(content_id: params.require(:lead_organisation)).id
```

Comment:
Is this the only usage of `org_id`? if so we can probably just have this as part of the join.

Reply:
Yep it's only used once and so I've added it to the main query.